src/test/java/org/apache/commons/configuration2/TestAbstractConfiguration.java

```diff
@@ -78,6 +78,49 @@ public void testAddPropertyDirect() {
         assertEquals(expected, list);
     }
 
+    @Test
+    void testContainsValue(){
```

Comment:
The style of the component is to use `public` test methods prefixed with "test" in the name and camel-case. Look at other tests for examples.

Reply:
Will correct.